doc/scenario.md

```diff
@@ -32,6 +34,62 @@ start(Id) ->
     %% send some messages again
     ok.
 ```
+or, using the `start/2` function:
+
+```erlang
+-module(my_scenario).
+
+-export([init/0]).
+-export([start/2]).
+
+init() ->
+    %% initialize some metrics
+    Settings = get_settings(),
+    {ok, Settings}.
+
+start(Id, Settings) ->
+    %% connect user using Settings
+    %% fetch user's history
+    %% send some messages
+    %% wait a little bit
+    %% send some messages again
+    ok.
+```
+
+For more advanced features, `amoc_scenario` behaviour proposes optional callbacks:
```

Comment:
Introducing a header here may make it more readable that a new, important section starts here.

Reply:
Is `Optional callbacks` ok? Or something like those `advanced features`?

Reply:
I vote for `advanced features` :)